### PR TITLE
enh(doc): prepare release notes for Centreon Gorgone 20.10.7

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Prepare release note for Centreon Gorgone 20.10.7 release

## Target version

- [x] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md [hunk 1]
```diff
@@ -1100,6 +1100,21 @@ with the –pool\_size X argument or -s X.
...
+Release date: `July 27, 2022`
+
+#### Bug fixes
+
+- Fixed an issue that caused Service Discovery scans to fail because the wrong message was caught.
+- Fixed an issue that could make Gorgone crash in pull mode.
+- Fixed an issue that prevented Gorgone from handling advanced [Service Discovery features](https://docs.centreon.com/21.10/en/monitoring/discovery/services-discovery.html#advanced-options) correctly.
+- Fixed an issue in the module management that could cause crashes.
+
+#### Améliorations
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md:1114]:
```suggestion
#### Enhancements
```

File: i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md [hunk 2]
```diff
@@ -1100,6 +1100,21 @@ with the –pool\_size X argument or -s X.
...
+
+- Added an "audit" module to Gorgone to provide an overview of the system status, package versions, + some Centreon metrics.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md:1116]:
```suggestion
- Added an "audit" module to Gorgone to provide an overview of the system status, package versions, and some Centreon metrics.
```

File: versioned_docs/version-20.10/releases/centreon-core.md [hunk 3]
```diff
@@ -1096,6 +1096,22 @@ with the –pool\_size X argument or -s X.
...
+#### Bug fixes
+
+- Fixed an issue that caused Service Discovery scans to fail because the wrong message was caught.
+- Fixed an issue that could make Gorgone crash in pull mode.
+- Fixed an issue that prevented Gorgone from handling advanced [Service Discovery features](https://docs.centreon.com/21.10/en/monitoring/discovery/services-discovery.html#advanced-options) correctly.
+- Fixed an issue in the module management that could cause crashes.
+
+#### Enhancements
+
+- Added an "audit" module to Gorgone to provide an overview of the system status, package versions, + some Centreon metrics.
```
Comment (smau-centreon) [inline versioned_docs/version-20.10/releases/centreon-core.md:1112]:
```suggestion
- Added an "audit" module to Gorgone to provide an overview of the system status, package versions, and some Centreon metrics.
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Comment by smau-centreon** [comment]
The "Enhancements" section should be placed before the "Bug fixes" one.

**Review by smau-centreon — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**